api/src/devcomp/infrastructure/datasources/learning-content/samples/elements/qrocm.sample.js

```diff
@@ -15,7 +15,7 @@ export function getQrocmSample() {
         type: 'input',
         inputType: 'text',
         size: 1,
-        display: 'inline',
+        display: 'block',
```

Comment:
Histoire d'avoir la démo des deux `display`

```suggestion
        display: 'inline',
```